content/pt-br/docs/concepts/configuration/configmap.md

```diff
@@ -0,0 +1,280 @@
+---
+title: ConfigMaps
+content_type: concept
+weight: 20
+---
+
+<!-- overview -->
+
+{{< glossary_definition term_id="configmap" prepend="Um ConfigMap é" length="all" >}}
+
+{{< caution >}}
+O ConfigMap não oferece confidencialidade ou encriptação.
+Se os dados que você deseja armazenar são confidenciais, utilize
+{{< glossary_tooltip term_id="secret" >}} ao invés de um ConfigMap,
+ou utilize ferramentas adicionais (de terceiros) para manter seus dados privados.
+{{< /caution >}}
+
+
+<!-- body -->
+## Motivação
+
+Utilize um ConfigMap para manter a configuração separada do código da aplicação.
+
+Por exemplo, imagine que você esteja desenvolvendo uma aplicação que pode ser executada
+no seu computador local (para desenvolvimento) e na nuvem (para manipular tráfego real).
+Você escreve código para ler a variável de ambiente chamada `DATABASE_HOST`.
+No seu ambiente local, você configura essa variável com o valor `localhost`. Na nuvem, você
+configura essa variável para referenciar um {{< glossary_tooltip text="serviço" term_id="service" >}}
+do Kubernetes que expõe o componente do banco de dados ao seu cluster.
+Isto permite que você baixe uma imagem de contêiner que roda na nuvem e depure o exato
```

Comment:
s/depure o exato/depure exatamente (wdyt?)

Reply:
Sounds better indeed! Just updated it!